src/Uno.UI.Runtime.Skia.Gtk/Input/GtkCorePointerInputSource.cs

```diff
@@ -596,4 +610,100 @@ private void RaisePointerWheelChanged(PointerEventArgs ptArgs, [CallerMemberName
 
 		PointerWheelChanged?.Invoke(this, ptArgs);
 	}
+
+	/// <summary>
+	/// The coordinates received in the event args are relative to the widget that raised the event.
```

Comment:
This is quite different from what the docs says 👀 :

https://docs.gtk.org/gdk3/struct.EventCrossing.html

![image](https://github.com/unoplatform/uno/assets/31348972/5b3256d0-0276-42f6-a16c-7f9b82906d21)

Reply:
@dr1rrb does it look familiar?

Reply:
Double checked now. I have no idea what the docs are about, but the x and y coordinates are affected depending on whether the event is on a random widget (e,g. gtk textbox) or on the canvas directly. Furthermore, I've hand-measured the numbers and they're indeed relative to the top-left corner of the widget of interest.